pom.xml

```diff
@@ -176,6 +176,12 @@
         <jodah-failsafe.version>0.9.5</jodah-failsafe.version>
         <apache.shiro.version>1.3.2</apache.shiro.version>
         <spark.version>2.5.5</spark.version>
+
+        <!-- gitflow-incremental-builder -->
+        <!-- See https://github.com/vackosar/gitflow-incremental-builder for a list of all the options -->
+        <gib.referenceBranch>refs/remotes/origin/master</gib.referenceBranch>
```

Comment:
My thinking here @shaundmorris was to use a default that would work for people most of the time if and when they use the incremental builder.

For our Jenkins jobs/pipelines, the job should always specify this value on the command line and override this default value. This way we never have to update the pom file.

We could always consider using `libero` or our release scripts to update this value for us when we create other branches if we feel it would be useful. But that wouldn't help with the remote name so even that wouldn't be a 100% solution.